src/main/java/com/cupid/jikting/member/controller/MemberController.java

```diff
@@ -45,9 +46,12 @@ public ResponseEntity<Void> update(@AuthorizedVariable Long memberProfileId, @Re
         return ResponseEntity.ok().build();
     }
 
-    @PatchMapping("/profile")
-    public ResponseEntity<Void> updateProfile(@AuthorizedVariable Long memberProfileId, @RequestBody MemberProfileUpdateRequest memberProfileUpdateRequest) {
-        memberService.updateProfile(memberProfileId, memberProfileUpdateRequest);
+    @PostMapping("/profile")
+    public ResponseEntity<Void> updateProfile(@AuthorizedVariable Long memberProfileId,
+                                              @RequestPart MultipartFile file,
+                                              @RequestPart MemberProfileUpdateRequest memberProfileUpdateRequest)
+            throws IOException {
```

Comment:
p3. 예외 던지는 곳의 개행을 지우는건 어떨까요?

```suggestion
                                              @RequestPart MemberProfileUpdateRequest memberProfileUpdateRequest) throws IOException {
```